geopandas/geodataframe.py

```diff
@@ -777,6 +789,42 @@ def _to_geo(self, **kwargs):
 
         return geo
 
+    def to_wkb(self):
+        """
+        Encode all geometry columns in the GeoDataFrame to WKB.
+
+        Returns
+        -------
+        DataFrame
+            geometry columns are encoded to WKB
+        """
+
+        df = DataFrame(self.copy())
+
+        # Encode all geometry columns to WKB
+        for col in df.columns[df.dtypes == "geometry"]:
+            df[col] = to_wkb(df[col].values)
+
+        return df
+
+    def to_wkt(self):
```

Comment:
```suggestion
    def to_wkt(self, **kwargs):
```

Similar as above. The original array method can pass kwargs to pygeos `to_wkt`. We should forward them.

Reply:
I have done this. I added a note to the docstring that the kwargs are only used if pygeos is installed, stating by implication that they aren't used with shapely. (Since with shapely, `wkt` and `wkb` are properties that can't take any args.) Let me know if that docstring isn't appropriate or consistent with the style or anything like that.

Reply:
Thanks. I think it is okay. At one point, pygeos will become shapely2.0 so this pygeos/shapely difference is only temporary.